client/src/main/java/io/pravega/client/stream/impl/ReaderGroupStateManager.java

```diff
@@ -405,7 +403,9 @@ private int calculateNumSegmentsToAcquire(ReaderGroupState state) {
 
     @VisibleForTesting
     static Duration calculateAcquireTime(String readerId, ReaderGroupState state) {
-        return TIME_UNIT.multipliedBy(state.getNumberOfReaders() - state.getRanking(readerId));
+        int multiplier = state.getNumberOfReaders() - state.getRanking(readerId);
+        Preconditions.checkArgument(multiplier >= TIME_UNIT.getSeconds(), "Invalid acquire timer multiplier");
```

Comment:
Shouldn't that be `Preconditions.checkArgument(multiplier >= 1, ...`

Reply:
In this case is the same, but I changed it to what you suggest.